CMakeLists.txt

```diff
@@ -73,6 +73,7 @@ include_directories(
     ./
     ../
     include/
+    3rdparty/
```

Comment:
You should rather move 'lib' folder content to 3rdparty folder, or move pwave to 'lib'.
Personally I would move 'lib' content to 3rdparty. 
Entia non sunt multiplicanda praeter necessitatem.